lighthouse-core/lib/page-functions.js

```diff
@@ -218,6 +218,52 @@ function getNodeSelector(node) {
   return parts.join(' > ');
 }
 
+/**
+ * @param {?HTMLElement} element
```

Comment:
why nullable? I don't believe it needs to be (maybe originally called recursively and no longer necessary?)